README.md

```diff
@@ -32,7 +32,9 @@ Example call: `approval-check <kerberos_id>`
 It writes to `new-trino-config-secret.yaml` in your current directory. This file can be used to raise a PR to the internal-data-hub repo. The script currently does not check for existing duplicate items, so be careful when adding new acl rules.
 
 Example create call: `trino-acl -a create -c schemas -u user -n test -r foo`
-Example delete call: `trino-acl -a delete -u group --type tables --name cost-management --resource-name costmgmt`
+Example delete call: `trino-acl -a delete -c tables -u group -n cost-management -r costmgmt`
```

Comment:
for documentation we should use the full flag text instead of `-a`, `-c` etc

Reply:
kay, im gonna update that now!